extensions/wikia/ArticleVideo/styles/video-attribution.scss

```diff
@@ -0,0 +1,42 @@
+@import 'skins/shared/color';
+@import 'skins/shared/mixins/flexbox';
+@import 'extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-variables/colors';
+
+.featured-video__attribution-container {
+	@include align-items(center);
+	@include flexbox();
+	@include justify-content(center);
+	border-bottom: 1px solid rgba($color-text, .5);
+	padding: 6px;
+}
+
+.featured-video__attribution-avatar {
+	height: 20px;
+	margin-right: 6px;
+	width: 20px;
+}
+
+.featured-video__attribution-username, .featured-video__attribution-icon {
```

Comment:
nitpicky, I'd throw \n after ,
it's easier to notice that these are two selectors